src/btree/bt_discard.c

```diff
@@ -245,20 +245,31 @@ __free_page_modify(WT_SESSION_IMPL *session, WT_PAGE *page)
 void
 __wt_ref_addr_free(WT_SESSION_IMPL *session, WT_REF *ref)
 {
+    WT_PAGE *home;
     void *ref_addr;
 
+    home = ref->home;
```

Comment:
np: I would put the comment right before this assignment, as the comment talks about why this is done and will flow better.

Reply:
[a441a1a](https://github.com/wiredtiger/wiredtiger/pull/8870/commits/a441a1a6c4e78a4a86298c0474520dc6f75a8ba5)